classes/email-templates/class-pmpro-email-template-billing-admin.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+class PMPro_Email_Template_Billing_Admin extends PMPro_Email_Template {
+
+	/**
+	 * The user object of the user to send the email to.
+	 *
+	 * @var WP_User
+	 */
+	protected $user;
+
+	/**
+	 * The {@link MemberOrder} object of the order that was updated.
+	 *
+	 * @var MemberOrder
+	 */
+	protected $order;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_User $user The user object of the user to send the email to.
+	 * @param MemberOrder $order The order object that is associated to the member.
+	 */
+	public function __construct( WP_User $user,  MemberOrder $order ) {
+		$this->user = $user;
+		$this->order = $order;
+	}
+
+	/**
+	 * Get the email template slug.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email template slug.
+	 */
+	public static function get_template_slug() {
+		return 'billing_admin';
+	}
+
+	/**
+	 * Get the "nice name" of the email template.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The "nice name" of the email template.
+	 */
+	public static function get_template_name() {
+		return __( 'Billing Information Updated (admin)', 'paid-memberships-pro' );
+	}
+
+	/**
+	 * Get "help text" to display to the admin when editing the email template.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The help text.
+	 */
+	public static function get_template_description() {
+		return __( 'Members can update the payment method associated with their recurring subscription. This email is sent to the site administrator as a confirmation of a payment method update.', 'paid-memberships-pro' );
+	}
+
+	/**
+	 * Get the default subject for the email.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The default subject for the email.
+	 */
+	public static function get_default_subject() {
+		return __( 'Billing information has been updated for !!user_login!! at !!sitename!!', 'paid-memberships-pro' );
+	}
+
+	public static function get_default_body() {
+		return __( '<p>The billing information for !!display_name!! at !!sitename!! has been changed.</p>
+<p>Account: !!display_name!! (!!user_email!!)</p>
+<p>
+	Billing Information:<br />
+	!!billing_name!!<br />
```

Comment:
This is the only template that seems to build out the address using individual variables. Is now an ok time to swap this to:

Billing Information:<br />
!!billing_address!!

Reply:
Done